metadata/aquatics-2021-05-01-BTW.yml

```diff
@@ -108,14 +108,14 @@ metadata:
       status: present #options: absent, present, data_driven, propagates, assimilates
       complexity: 2
       #Leave everything below blank if status = absent, present, or data_driven
-      propagation:
-        type: ensemble #how does your model propogate parameter uncertainity?
-        size: 2000
-      #Leave everything below blank  UNLESS status = assimilates
-      assimilation:
-        type: refit
-        reference: "none"
-        complexity: 4
+      # propagation:
+      #   type: ensemble #how does your model propogate parameter uncertainity?
+      #   size: 2000
+      # #Leave everything below blank  UNLESS status = assimilates
+      # assimilation:
+      #   type: refit
+      #   reference: "none"
+      #   complexity: 4
```

Comment:
don't need these lines since the parameters are present but not drawn from a distribution / propogated into the forecasts. The may be data driven , but I'm not sure if the model was fit to the POSE data before running the forecast. If so, change this to `data_driven`